ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -507,6 +507,11 @@ def test_source_ipahealthcheck_dogtag_ca_dogtagcertsconfigcheck(self):
         Testcase checks behaviour of check DogtagCertsConfigCheck in
         ipahealthcheck.dogtag.ca when tomcat config file is removed
         """
+        version = tasks.get_tasks.get_pki_version(self.master)
+        if parse_version(version) >= parse_version("11.5"):
+            pytest.skip("Skipping test for 11.5 pki version, since the "
+                        "check CADogtagCertsConfigCheck is not present "
+                        "in source 'pki.server.healthcheck.meta.csconfig")
```

Comment:
The comment is not accurate. It should read:
```
            pytest.skip("Skipping test for 11.5 pki version, since the "
                        "check DogtagCertsConfigCheck itself is skipped. "
                        "See ipa-healthcheck ticket 317.")
```